pallets/nfts/src/lib.rs

```diff
@@ -220,8 +206,8 @@ pub mod pallet {
     #[pallet::generate_deposit(pub(super) fn deposit_event)]
     #[pallet::metadata(T::AccountId = "AccountId", T::NFTId = "NFTId")]
     pub enum Event<T: Config> {
-        /// A new NFT was created. \[nft id, owner\]
-        Created(T::NFTId, T::AccountId),
+        /// A new NFT was created. \[nft id, owner, series id\]
+        Created(T::NFTId, T::AccountId, NFTSeriesId),
```

Comment:
Shall we just include `NFTDetails` instead?

Reply:
Maybe we should wait until we are done with the "capsule" feature to see what is needed or not.

Reply:
Up to you guys - I will let @ybensacq have the last word on this

Reply:
Ok we will see that in the capsule branch